ext/SymbolicsLuxCoreExt.jl

```diff
@@ -0,0 +1,17 @@
+module SymbolicsLuxCoreExt
+
+using LuxCore, Symbolics
+
+@register_array_symbolic LuxCore.partial_apply(
+    model::LuxCore.AbstractExplicitContainerLayer, x::AbstractArray, ps::Union{NamedTuple, <:AbstractVector}, st::NamedTuple) begin
+    size = LuxCore.outputsize(model[end])
```

Comment:
`model[end]` is not well defined, you need to query the output size of the entire `model`. Consider:

`Parallel(vcat, Dense(2 => 2), Dense(2 => 3))`, even if you had a sense of "end" the individual layers can't reason about the output size of the parent layer

Reply:
Essentially if done right you shouldn't need this dispatch at all, just the `AbstractExplicitLayer` one should be enough

Reply:
Should `outputsize` be defined recursively in Lux?

Reply:
you will have to rely on users providing the correct outputsize function, it is quite hard to do it in the general case, since you need to trace all the internal operations (in principle if you have outputsize for all primitives Symbolics should be able to compute the output size of the Chain/Parallel or any container layer).

But for now, I would recommend defining the outputsize of the common Lux layers in containers.jl

Reply:
Yes, let's just rely on the interface and not try to go around it. So then I presume that means `size = LuxCore.outputsize(model)` and if the user needs to specify this function they will get an error and that's fine. I'd rather stay within the standard API rather than try to do something hacky that only works partially.